### PR TITLE
docs(ha): Add paragraph indicating that Redis endpoint must include protocol and port.

### PR DESCRIPTION
**Pull request by joonlim**

### CODE REVIEW

**Comment by joonlim** [comment]
@dorbin @jtk54 @lwander

File: reference/halyard/high-availability.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ Although by default the three Clouddriver services will communicate with the glo
 hal config deploy ha clouddriver edit --redis-master-endpoint $REDIS_MASTER_ENDPOINT --redis-slave-endpoint $REDIS_SLAVE_ENDPOINT
 ```
 
+The values for `REDIS_MASTER_ENDPOINT` and `REDIS_SLAVE_ENDPOINT` must be a valid full Redis endpoint, not just the host name. This usually means that the endpoints should start with a `redis://` protocol and end with a `:6379` port. An example could be `redis://127.0.0.1:6379`.
```
Comment (dorbin) [inline reference/halyard/high-availability.md]: s/a valid full Redis enpoint/valid full Redis endpoints/

s/the hostname/hostnames/

s/An example code be/For example:/

Comment (dorbin) [reply reference/halyard/high-availability.md]: "...usually means...should start with..."

Are there cases we support where the endpoint won't start with `redis://` and end with that port.

**Review by dorbin — CHANGES_REQUESTED** [review]

**Comment by joonlim** [comment]
@dorbin Updated to direct to doc describing redis URI standard

**Review by dorbin — APPROVED** [review]

**Pull request merged**